File: src/main/java/core/data/attribute/Switch.java
package core.data.attribute;

/**
 * Switch class is used to indicate {@link DataAttribute} that holds no value: it's either declared in attributes or no
 */
//TODO: (avlomakin) consider creation superclass for attributes with and without inner values
public final class Switch {

    public static final Switch instance = new Switch();

    private Switch(){

    }
}

File: src/main/kotlin/com/mirf/core/data/attribute/Switch.kt
package com.mirf.core.data.attribute

/**
 * Switch class is used to indicate [DataAttribute] that holds no value: it's either declared in attributes or no
 */
//TODO: (avlomakin) consider creation superclass for attributes with and without inner values
object Switch{
    fun get() : Instance = Instance()

    class Instance internal constructor()
}
